modules/products/bamboo/helm.tf

```diff
@@ -1,7 +1,11 @@
 # Install helm chart for Bamboo Data Center.
 
 resource "helm_release" "bamboo" {
-  depends_on = [kubernetes_job.import_dataset]
+  depends_on = [
+    kubernetes_job.import_dataset,
+    module.nfs,
```

Comment:
Do we need to have `module.nfs` here? `time_sleep` is already depends on nfs, so here we can remove it as is embedded.

Reply:
Nice catch. Yes, depending on nfs module is implied here. Leftovers from trying just to get away with dependency on nfs (without sleep)